variables.tf

```diff
@@ -126,8 +126,8 @@ variable "placement" {
 
 variable "credit_specification" {
```

Comment:
```suggestion
variable "cpu_credits" {
```

for consistency, let's name it the same as the module's attribute

Reply:
Done!